src/dnsperf.c

```diff
@@ -480,6 +486,11 @@ setup(int argc, char** argv, config_t* config)
     perf_opt_add('v', perf_opt_boolean, NULL,
         "verbose: report each query and additional information to stdout",
         NULL, &config->verbose);
+    perf_long_opt_add("--doh-uri", perf_opt_string, "doh_uri",
```

Comment:
This is not long opts in that way, you need to remove `--`. It's then `-O doh-uri=<value>`.